src/Nethermind/Nethermind.Runner/Nethermind.Runner.csproj

```diff
@@ -67,10 +67,6 @@
       <Link>Nethermind Ethereum Client - Terms and Conditions 20190831.pdf</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="..\Chains\**\*.*">
-      <Link>chainspec\%(RecursiveDir)%(Filename)%(Extension)</Link>
```

Comment:
Have you checked if this is included in the package?

Reply:
it is included here
https://github.com/NethermindEth/nethermind/pull/4450#diff-84ee305ac5d2077574b4d1ba10854b6d1d430062b2f63de10adcc2fff2a778ee